app/backend/templates/host_reference.md

```diff
@@ -15,7 +15,7 @@ This is what they wrote:
 
 Thanks for using Couchers to organize this interaction! We hope you had an enjoyable, fulfilling time.
 {% else %}
-Please go and write a reference for them too. It's a nice gesture and helps us build a community together! When you've both written a reference, they'll both become visible, otherwise {{ escape(reference.from_user.name) }}'s reference will become visible 2 weeks after the end of your interaction.
+Please go and write a reference for them too. It's a nice gesture and helps us build a community together! When you've both written a reference, the references will both become visible. Otherwise {{ escape(reference.from_user.name) }}'s reference will become visible 2 weeks after the end of your interaction, after which you cannot write a reference back.
```

Comment:
Sorry, reading this again,

>When you've both written a reference, the references will both become visible.

-->Once you've written a reference, both references will become visible.

Since their reference has already been written

>after which you cannot write a reference back

Nice addition